piwheels/slave/builder.py

```diff
@@ -239,7 +268,10 @@ def dependencies(self):
         dependency system (e.g. "apt", "pip", etc.) to set of package names for
         that system.
         """
-        return self._dependencies
+        return {
+            'apt': sorted(self.apt_dependencies),
+            'pip': sorted(self.pip_dependencies),
+        }
```

Comment:
Why are we sorting this?

Reply:
Looks like we were before: https://github.com/piwheels/piwheels/blame/main/piwheels/slave/builder.py#L544 - this bit is just moved now